### ISSUE TITLE
[UI idea] Improvements to adventures reordering

### ISSUE THREAD

**Issue by jpelay** [hedyorg/hedy#3955]
**Idea incl level**
After #3854 is merged we need to address these improvements:

Originally posted by @Felienne 
>    Now that we have the new layout, I think we can also move quiz and puzzle into the "dragger" instead of a tickbox below?
    We need to change the text for the enable/disable buttons (I can do that)
    If there are no adventures available, can we remove the drowdown? And empty one is a bit weird. We can even consider a button there with "add you own"
    We now have 2 places for all levels (in addition to the dragger) the top enable/disable and the dates list. Can we make that into one? Maybe just the date lists with a tick mark?

Also we need to add a Cypress test for the new drag & drop functionality!

**Additional context**
Add any other context about the idea here.

**Comment by Felienne** [comment]
Hi @Annelein!

Maybe many of these (or all?) have been fixed already? If so feel free to close right away.

**Comment by Annelein** [comment]
@jpelay I don't think there is an Cypress test for dragging the adventures right? 

And then all thats left is this: _`If there are no adventures available, can we remove the drowdown? And empty one is a bit weird.`_

**Comment by jpelay** [comment]
> @jpelay I don't think there is an Cypress test for dragging the adventures right?
No, we used to use our own solution for the dragging and dropping, but we later switched to a library and it doesn't work welk with Cypress.

> And then all thats left is this: _`If there are no adventures available, can we remove the drowdown? And empty one is a bit weird.`_

I think I can tackle this in #5540 or maybe in another PR but using that same element.

**Comment by jpelay** [comment]
> @jpelay I don't think there is an Cypress test for dragging the adventures right?

No, we used to use our own solution for the dragging and dropping, but we later switched to a library and it doesn't work welk with Cypress.
 
> And then all thats left is this: _`If there are no adventures available, can we remove the drowdown? And empty one is a bit weird.`_

I think I can tackle this in #5540 or maybe in another PR but using that same element.

**Comment by Annelein** [comment]
@jpelay allright! Should I just create a new issue for removing the dropdown?
And also a new issue for testing the drag and drop for in the future?

**Comment by Annelein** [comment]
> [@jpelay](https://github.com/jpelay) allright! Should I just create a new issue for removing the dropdown? And also a new issue for testing the drag and drop for in the future?
> 
> Then we can close this one.

@jpelay 👆🏼
